### DISCUSSION TITLE
Remove unused CSS in the website

### DISCUSSION THREAD

**Discussion by srv2309** [vercel/next.js#15464] [Help]
I have been testing a website https://jobdiscovers.com/ build in WordPress where the Pagespeed is showing **Remove unused CSS** from the website. That particular CSS is actually the minified version of all the CSS and is preloaded in the header.

How can I eliminate the issue

**Comment by jamesmosier** [comment]
This doesn't really sound related to Next.js, but you may find more help by looking elsewhere. This article looks useful https://css-tricks.com/how-do-you-remove-unused-css-from-a-site/
